Makefile

```diff
@@ -8,12 +11,29 @@ build: gofmtcheck golint vet
 install:
 	go install -ldflags "${LDFLAGS}"
 
+# unit tests
+# usage:
+# - run all the unit tests: make test
+# - run some particular test: make test TEST_ARGS="-run TestAccLibvirtDomain_Cpu"
 test:
-	go test -v -covermode=count -coverprofile=profile.cov ./libvirt
+	go test -v $(TEST_ARGS_DEF) $(TEST_ARGS) ./libvirt
 	go test -v .
+
+# acceptance tests
```

Comment:
nice one here. can you update also this doc: https://github.com/dmacvicar/terraform-provider-libvirt/blob/master/CONTRIBUTING.md#running-the-tests-testacc

Reply:
Done.